server/data/quotes.json

```diff
@@ -162,5 +162,9 @@
   {
     "sentence": "For last year's words belong to last year's language. And next year's words await another voice.",
     "author": "T.S. Eliot, Four Quartets"
+  },
+  {
+    "sentence": "Sometimes it pays to stay in bed on Monday, rather than spending the rest of the week debugging Monday’s code.",
```

Comment:
Hey @paulzay thanks for opening a PR!

Hope you can make the quote about learning a foreign language instead. :)

Reply:
Done